rfcs/48-diverse-defaults.md

```diff
@@ -0,0 +1,60 @@
+# Feature Name: Different plugin groups to support different use cases (diverse-defaults for short)
+
+## Summary
+
+There are many different categories of bevy applications. Each use case has a sufficiently different commonsense default configuration to warrant different use case specific default plugins.
+
+## Motivation
+
+Bevy would be doing this to support easier and faster 'getting started' user story for non-minimal bevy applications. It will be particularly helpful to beginners but would also be a nice ergonomics/UX win for experienced bevy devs to not have to configure every minute detail manually when setting up a new project that is in one of the above outlined application domains.
+
+## User-facing explanation
+
+Where there used to just be `DefaultPlugins` and `MiniminalPlugins` there will now be many different `DefaultPlugins` a user can choose from that exist for many common categories of bevy applications.
+
+Bevy now provides you a number of convenient `PluginGroups` to help you get your application started with sensible defaults for your use case. The current list is:
```

Comment:
I would really like "turn-based game" as an plugin group starter.

Reply:
How would that differ from a regular 2d starter? I think there's a risk of overscoping and complicating this.

Reply:
I second @speakk's sentiment and question.

Reply:
Yeah, I'm totally wrong here.